go.sum

```diff
@@ -41,6 +41,7 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/stretchr/objx v0.1.0 h1:4G4v2dO3VZwixGIRoQ5Lfboy6nUhCyYzaqnIAPPhYs4=
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/testify v1.3.0/go.mod h1:M5WIy9Dh21IEIfnGCwXGc5bZfKNJtfHm1UVUgZn+9EI=
```

Comment:
what did we do to pull in this version?

Reply:
Great question. Will investigate.

Reply:
I'm kinda baffled by this. All the functions I used that're part of the `assert` package appear to be contained in the latest Testify version. If I'm understanding the output of `go mod why` correctly, it looks like `stretchr/objx` requires `testify` so many that's why? But I can't figure out what we're using `objx` for if it's not just a dep of `testify`...